packages/@okta/vuepress-site/use_cases/inline_hooks/index.md

```diff
@@ -121,7 +122,37 @@ The names of commands follow Java-style reverse DNS name format, beginning with
 
 Lets you return error messages. How the error data is used varies by inline hook type.
 
-Within an `error` object, you need to provide an `errorSummary` property set to a text string. Additionally, you can use an `errorCauses` object to supply more information. A single error object can contain multiple `errorCauses` objects. The fields within errorCauses are: `errorSummary`, `reason`, `locationType`, `location`, and `domain`.
+The `error` object should have the following structure:
```

Comment:
There a couple hook pages (Import Hooks and Token Hooks) that mention the error object. I would leave whatever use-case specific information they contain about errors, but would remove the descriptions of the error object structure they have and link to this page instead.